### DISCUSSION TITLE
LiteLLM for enhancing authoring experiences

### DISCUSSION THREAD

**Discussion by rishi-raj-jain** [BerriAI/litellm#1879] [Show and tell]
Wrote a blog post on how to create a custom CMS using Xata, Remix, Novel, LiteLLM, and Vercel 👇🏻

https://xata.io/blog/content-management-system-remix-xata

**Comment by ishaan-jaff** [comment]
How did you get litellm in js @rishi-raj-jain ?

**Reply by rishi-raj-jain** [reply]
I'm using the litellm package: https://github.com/rishi-raj-jain/remix-wysiwyg-litellm-xata-vercel/blob/master/app/routes/api_.generate.tsx#L1

**Reply by rishi-raj-jain** [reply]
https://www.npmjs.com/package/litellm
